libs/analysis/aframe/analysis/ledger/events.py

```diff
@@ -50,6 +50,21 @@ def significance(self, threshold: F, T: float) -> F:
         nb = self.nb(threshold)
         return 1 - np.exp(-T * (1 + nb) / self.Tb)
 
+    def apply_vetos(self, vetos: np.ndarray):
+        # TimeSlideEventSet does not have shift information,
+        # so apply vetoes as if the shift is zero
+        mask = np.logical_and(
+            vetos[:, :1] < self.time, vetos[:, 1:] > self.time
+        )
+
+        # mark a background event as vetoed
+        # if it falls into _any_ of the segments
+        veto_mask = False
+        veto_mask |= mask.any(axis=0)
```

Comment:
Am I missing something with why this would be any different than just setting `veto_mask = mask.any(axis=0)`?

Reply:
Nope. I think I copy pasted from somewhere without any thought.